Cargo.toml

```diff
@@ -35,6 +39,7 @@ tests-gmp = ["rug/std"]
 tests-glass-pumpkin = ["glass_pumpkin"]
 tests-exhaustive = []
 tests-all = ["tests-openssl", "tests-gmp", "tests-exhaustive", "tests-glass-pumpkin"]
+rayon = ["dep:rayon"]
```

Comment:
Do you think we should expose the specifics of how we do parallelization, or just call the feature `multicore` or something? Does the choice of the thread pool affect the user?

Reply:
I like "multicore"!